command/forceleave/forceleave.go

```diff
@@ -52,8 +55,9 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
-	if c.prune {
-		err = client.Agent().ForceLeavePrune(nodes[0])
+	if c.prune || c.datacenter != "" {
```

Comment:
This should be adjusted after removing `prune` from `QueryOptions`

Reply:
```
if c.prune || c.datacenter != "" {
    err = client.Agent().ForceLeaveWithOpts(nodes[0], c.datacenter, c.prune)
} else {
    err = client.Agent().ForceLeave(nodes[0])
}
```
Which way should it be adjusted? `ForceLeaveWithOpts` can be with unspecified prune and specified datacenter and vice versa, in other case it's just `ForceLeave`

Reply:
That block you just posted is what I had in mind.